src/index.html

```diff
@@ -6,8 +6,43 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
     <h1>Product cards</h1>
+
+    <div class="card" data-qa="card">
+      <img
+        class="card__picture"
+        src="images/imac.jpeg"
+        alt="product"
+      >
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__description">
+        Product code: 195434
+      </p>
+      <div class="card__inner">
```

Comment:
Add the blank line between the blocks
```suggestion
      >
      
      <h2 class="card__title">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h2>
      
      <p class="card__description">
        Product code: 195434
      </p>
      
      <div class="card__inner">
```